### DISCUSSION TITLE
Default data-elb prefix

### DISCUSSION THREAD

**Discussion by alexanderkirtzel** [elbwalker/walkerOS#59] [Announcements]
One thing I love about #opensource development is the sincere, helpful feedback you get. And it’s a lot!
We have become significantly faster in development. Mostly because people help us to prioritize features & issues.

We could have wasted a lot of resources to make the walker.js more helpful “in our opinion”.
But we’ve learned that many developers felt uncomfortable with our `elb-attributes` and wished for a valid `data-` prefix. A small data-prefix that kept Devs from using the walker.js 🧐

We didn’t get that many helpful responses while not being open-source. And it's just one example.

In short: the default prefix is now `data-elb` & thanks for your feedback 🥳
